2024-07-11 18:18:50,118 INFO    MainThread:46121 [wandb_setup.py:_flush():71] Unhandled environment var: WANDB__SERVICE_WAIT
2024-07-11 18:18:50,118 INFO    MainThread:46121 [wandb_setup.py:_flush():71] setting env: {'mode': 'online'}
2024-07-11 18:18:50,119 WARNING MainThread:46121 [wandb_setup.py:_flush():71] could not save program above cwd: /home/gauffrea/.local/lib/python3.8/site-packages/ray/_private/workers/default_worker.py
2024-07-11 18:18:50,119 INFO    MainThread:46121 [wandb_setup.py:_flush():71] setting login settings: {}
2024-07-11 18:18:50,119 INFO    MainThread:46121 [wandb_init.py:_log_setup():371] Logging user logs to /home/gauffrea/Projects/SMABalancingGroups/wandb/run-20240711_181850-57x2erv9/logs/debug.log
2024-07-11 18:18:50,119 INFO    MainThread:46121 [wandb_init.py:_log_setup():372] Logging internal logs to /home/gauffrea/Projects/SMABalancingGroups/wandb/run-20240711_181850-57x2erv9/logs/debug-internal.log
2024-07-11 18:18:50,119 INFO    MainThread:46121 [wandb_init.py:init():404] calling init triggers
2024-07-11 18:18:50,119 INFO    MainThread:46121 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'wandb_project': 'SMA_selector', 'output_dir': 'outputs', 'slurm_output_dir': 'slurm_outputs', 'data_path': 'data', 'slurm_partition': None, 'max_time': 4320, 'num_hparams_seeds': 20, 'selector': 'mean_grp_acc_va', 'method': 'subg', 'dataset': 'SMA', 'init_seed': 0, 'hparams_seed': 0, 'eval_every_n_epochs': 4, 'arch': 'resnet18', 'n_HBO_runs': 3, 'n_eval_init_seed': 3, 'num_epochs': 5, 'batch_size': 25, 'eta': 0.1, 'lr': 0.0022825343088702303, 'weight_decay': 0.08632012725909878, 'up': 20, 'T': 3, 'scheduler': False, 'new_reset': False, 'config_name': 'configTime1.yaml', 'n_gpus': 1, 'IR': 0.30000000000000004, 'group': 'K=4_subg', 'group_best': 'mean_grp_acc_va_texture-dtd_K=4_subg_mu=0.1_seed=1', 'mode': 'classic', 'img_size': 256, 'split_seeds': [1], 'names': ['texture-dtd'], 'Ks': [4], 'mus': [0.1], 'methods': ['subg'], 'name': 'texture-dtd', 'K': 4, 'mu': 0.1, 'split_seed': 1}
2024-07-11 18:18:50,119 INFO    MainThread:46121 [wandb_init.py:init():449] starting backend
2024-07-11 18:18:50,120 INFO    MainThread:46121 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-07-11 18:18:50,132 INFO    MainThread:46121 [backend.py:ensure_launched():199] starting backend process...
2024-07-11 18:18:50,143 INFO    MainThread:46121 [backend.py:ensure_launched():204] started backend process with pid: 46228
2024-07-11 18:18:50,144 INFO    MainThread:46121 [wandb_init.py:init():458] backend started and connected
2024-07-11 18:18:50,146 INFO    MainThread:46121 [wandb_init.py:init():519] updated telemetry
2024-07-11 18:18:50,168 INFO    MainThread:46121 [wandb_init.py:init():549] communicating current version
2024-07-11 18:18:50,649 INFO    MainThread:46121 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.17.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-07-11 18:18:50,649 INFO    MainThread:46121 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2024-07-11 18:18:50,881 INFO    MainThread:46121 [wandb_init.py:init():592] starting run threads in backend
2024-07-11 18:18:51,972 INFO    MainThread:46121 [wandb_run.py:_console_start():1816] atexit reg
2024-07-11 18:18:51,972 INFO    MainThread:46121 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2024-07-11 18:18:51,972 INFO    MainThread:46121 [wandb_run.py:_redirect():1695] Redirecting console.
2024-07-11 18:18:51,973 INFO    MainThread:46121 [wandb_run.py:_redirect():1751] Redirects installed.
2024-07-11 18:18:51,973 INFO    MainThread:46121 [wandb_init.py:init():619] run started, returning control to user process
2024-07-11 18:18:57,083 INFO    MainThread:46121 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2024-07-11 18:18:57,084 INFO    MainThread:46121 [wandb_run.py:_restore():1758] restore
2024-07-11 18:19:00,006 INFO    MainThread:46121 [wandb_run.py:_restore():1758] restore
